tests/baselines/reference/looseThisTypeInFunctions.js

```diff
@@ -67,7 +68,7 @@ c.explicitVoid = c.explicitThis; // error, 'void' is missing everything
 var o = {
     n: 101,
     explicitThis: function (m) {
-        return m + this.n.length; // ok, this.n: any
+        return m + this.n.length; // error, 'length' does not exist on 'number'

```

Comment:
What I mean is, you run the tests, find out something is wrong, realize you have to update the test itself as well because the comment is now lying to you, and also update the baselines as well.